Content.Server.Database/Model.cs

```diff
@@ -385,6 +390,15 @@ public class Trait
         public string TraitName { get; set; } = null!;
     }
 
+    public class Loadout
+    {
+        public int Id { get; set; }
+        public Profile Profile { get; set; } = null!;
+        public int ProfileId { get; set; }
+
+        public string LoadoutName { get; set; } = null!;
+    }
+
```

Comment:
This looks odd and I assume copied from the other ones but:
1. Why not sealed
2. Why the nullable suppression.

Reply:
1. The others aren't sealed
2. It complains the variables are uninitialized, the others do this too